app/controllers/invite_controller.rb

```diff
@@ -82,16 +83,16 @@ def submit
         @type = "danger"
       else
         tester = Spaceship::Tunes::Tester::External.new({
-          'emailAddress' => {'value' => email},
```

Comment:
the `Spaceship::Tunes::Tester::External`  has been adopted to the new tester-api, which returns a different json, therefore in this case we need to craft it with the new format, i guess finding another way is the long-run solution.